### PR TITLE
Hw 5

### LINKED ISSUE #12: HW #5
- In every page of HTML, there can only be one ID assigned to an element. IDs are and should be unique to the page. However, class names can be assigned to as many html elements as you want. 
Please study a little: https://css-tricks.com/the-difference-between-id-and-class/
- ~**_OCAD: Create a new file called post-5.html. Add links to the posts.html page for both post-4 and post 5. Copy and paste your entire post-4.html file. Rename the h1 div to "Sellbrite Team Remix."_**~
- ~OCAD 2: For this file, remove all html style tags from the name column **_except for me and Homan_**. So remove the~ 

> <style="color:red, font: 14px"> 

~**_for everyone except me and Homan._**~

- ~In your post-4.html file, please assign a class of "engineering" to anyone who has a name that is part of the engineering team. Do the same for product, marketing, customer success, design etc.~
- OCAD 3: In the style tag, please style any td with the class of "engineering" as color "red," "customer success" yellow, "sales" green, "product" blue, "marketing" brown, "design" pink.
- It is possible that after you added all these classes and styles, that only the "customer success" class is not working correctly. **Hint**: the "customer success" class has two words instead of one like the others.
- Add the class of "leadership" on any td tags under the leadership column for anyone who is leadership. Using the style tag, change the color of the font to red if they have the class leadership (so the word "true" should come out as red, and false will remain black)
- For your name, ALSO put the id "special." (meaning you have both an id and a class). In the style tag, assign the "special" id to your favorite color: purple. What happens? Why do you think the ID takes priority over the class color?
- You might notice that me and Homan's names did not change color after you made the appropriate changes. Why do you think this is? What is the difference? The answer to this question and why your name is purple instead of yellow like the other cs members can be found here: http://tutorials.jenkov.com/css/precedence.html
- We often use the padding style attribute to move elements away from its border. Please study a little bit here: https://www.w3schools.com/cssref/pr_padding.asp
- For all td, do a padding of 8px and see what happens. 

Screen Shot 2020-03-25 at 3 27 13 PM

- Make PR and screenshot inside of the PR when done.
- Good luck!

### PR DESCRIPTION
**Pull request by teresa210**
image

### CODE REVIEW

File: posts/post-5.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+            <style>
+                .product {
+                    color: blue;
+                }
+                .customer {
+                    color: yellow;
+                }
+                .success {
+                    color: yellow;
+                }
```
Comment (hobertro) [inline posts/post-5.html]: this should be one word

File: posts/post-5.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <div>
+            <h1>Sellbrite Team Remix</h1>
+            <hr>
+
+            <style id="sb-team">
```
Comment (hobertro) [inline posts/post-5.html]: this should be on the div haha

Comment (teresa210) [reply posts/post-5.html]: made changes to this. Is it correct?

<div id="sb-team">
            <h1>Sellbrite Team Remix</h1>
            <hr>
        </div>
            <style>
                table, th, td {
                border: 1px solid black;
                border-collapse: collapse;
                }

File: posts/post-5.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        </div>
+            <style>
+                table, th, td {
+                border: 1px solid black;
+                border-collapse: collapse;
+                }
+                .product {
+                    color: blue;
+                }
+                .customer, .success {
```
Comment (hobertro) [inline posts/post-5.html]: still incorrect, we want one word for the class

**Pull request merged**